test/es-module/test-esm-loader-modulemap.js

```diff
@@ -24,11 +24,11 @@ const jsonModuleJob = new ModuleJob(loader, stubJsonModule.module,
                                     () => new Promise(() => {}));
 
 
-// ModuleMap.set and ModuleMap.get store and retrieve module jobs for a
-// specified url/type tuple; ModuleMap.has correctly reports whether such jobs
+// LoadCache.set and LoadCache.get store and retrieve module jobs for a
+// specified url/type tuple; LoadCache.has correctly reports whether such jobs
 // are stored in the map.
```

Comment:
Assuming we keep the resolve cache (rather than collapse them back into one cache per my other comment) then shouldn’t we have similar tests for `ResolveCache.set` and `setLazy` and `getSerialized` and so on?